Sources/StreamVideo/Controllers/CallController.swift

```diff
@@ -29,7 +29,7 @@ class CallController: @unchecked Sendable {
         }
     }
 
-    weak var call: Call? {
+    @PublishedWeak var call: Call? {
```

Comment:
Feels a bit strange to introduce published to a non-observable class. I'm not sure we need the wrapper, let's discuss it a bit more.

Reply:
As we discussed, we are going to remove this one as it doesn't add much value. Thanks